metaflow/runtime.py

```diff
@@ -591,6 +592,50 @@ def execute(self):
                 "The *end* step was not successful by the end of flow."
             )
 
+    def _run_exit_hooks(self):
+        try:
+            exit_hook_decos = self._flow._flow_decorators.get("exit_hook", [])
+            if not exit_hook_decos:
+                return
+
+            successful = ("end", ()) in self._finished or self._clone_only
+            pathspec = f"{self._graph.name}/{self._run_id}"
+            flow_file = self._environment.get_environment_info()["script"]
+
+            def _call(fn_name):
+                try:
+                    result = (
+                        subprocess.check_output(
+                            args=[
+                                "python",
```

Comment:
prefer something like `sys.executable`.  In Conda, that would be the conda executable, if that is not what you want, you can use `ENV_ESCAPE_PY` to get the outer python.